unit/java_bytecode/java_bytecode_parse_generics/class_bound.java

```diff
@@ -0,0 +1,8 @@
+class C{
```

Comment:
To be consistent with the naming, it would be great if you could rename the file to `BoundedGenericClasses.java`.

Reply:
ok, renamed source file